app.yaml

```diff
@@ -37,6 +37,8 @@ modules:
         - account: dollar
           permissions: [ burner, minter ]
         - account: dollar/yield
+        - account: warp
+          permissions: [ burner, minter ]
```

Comment:
Because we aren't enabling the synthetic token routes, I'm curious if we can simply remove the permissions? Additionally, in the Hyperlane SimApp, they do specify the `hyperlane` module account here, but wondering if that is necessary?